charts/timescaledb-single/templates/svc-prometheus.yaml

```diff
@@ -0,0 +1,21 @@
+# This file and its contents are licensed under the Apache License 2.0.
+# Please see the included NOTICE for copyright information and LICENSE for a copy of the license.
+#
+# This service is only created if Prometheus is enabled.
+{{ if .Values.prometheus.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "clusterName" . }}-prometheus
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ template "timescaledb.fullname" . }}
+  annotations:
+    prometheus.io/scrape: "true"
+    prometheus.io/port: "9187"
+spec:
+  ports:
+    - port: 9187
+  selector:
```

Comment:
This will connect to a random member of the ReplicaSet(?),
Should we also add `role: master` to the Selector?

Reply:
I believe it connects to all of them actually, since I get metrics for all the replicas and the master. That is probably useful rather than just the master

Reply:
:+1: